external/fv3fit/fv3fit/train.py

```diff
@@ -181,7 +181,7 @@ def main(args, unknown_args=None):
     os.makedirs("artifacts", exist_ok=True)
     logging.basicConfig(
         level=logging.INFO,
-        format="%(asctime)s [%(levelname)s] %(message)s",
+        format="%(asctime)s [%(levelname)s] %(filename)s::L%(lineno)d : %(message)s",
```

Comment:
Is this OK to change? Should I revert it? I really like being able to see where log lines are coming from.